cfe_internal/enterprise/CFE_hub_specific.cf

```diff
@@ -547,6 +547,27 @@ bundle agent cfe_internal_refresh_hosts_view
 
 }
 
+bundle agent cfe_internal_clear_last_seen_hosts_logs
+# @brief Clear last seen hosts log every 5 minutes
+{
+  meta:
+
+    (policy_server|am_policy_hub).enterprise_edition::
+
+      "tags" slist => { "enterprise_maintenance" };
+
+  commands:
+
+    (policy_server|am_policy_hub).enterprise_edition::
+
+      "$(sys.workdir)/httpd/php/bin/php" -> { "ENT-3550" }
+        args => "$(cfe_internal_hub_vars.docroot)/index.php cli_tasks clearLastSeenHostsLogs",
+        comment => "This clears the __lastseenhostslogs table. If the __lastseenhostslogs table is not cleared then it will contain unnecessary data.",
```

Comment:
If it's not cleaned up will it affect the ability to identify hosts that are duplicated in any way? If so please include that in the comment, if not then looks good.

Reply:
If it's not cleaned up you still have the ability to identify same hosts.